plugins/orchestrator-backend/package.json

```
@@ -56,22 +56,22 @@
     "export-dynamic": "janus-cli package export-dynamic-plugin"
   },
   "dependencies": {
-    "@backstage/backend-app-api": "^0.5.8",
-    "@backstage/backend-common": "^0.19.8",
-    "@backstage/backend-plugin-api": "^0.6.6",
+    "@backstage/backend-app-api": "^0.5.10",
+    "@backstage/backend-common": "^0.20.1",
+    "@backstage/backend-plugin-api": "^0.6.9",
     "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.2-janus.5",
```

```suggestion
    "@backstage/backend-dynamic-feature-service": "0.1.0"
```

Of course the import in the code should be changed as well.
This package is now part of the upstream backstage since Backstage 1.22.0.